Packs/PAN-OS/Playbooks/PAN-OS_-_Extract_IPs_From_Traffic_Logs_To_Sinkhole.yml

```diff
@@ -0,0 +1,922 @@
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "74":
+    id: "74"
+    taskid: 2688899c-a918-480c-8e2c-bdea84e79512
+    type: regular
+    task:
+      id: 2688899c-a918-480c-8e2c-bdea84e79512
+      version: -1
+      name: Retrieve only IPs from denied requests to sinkhole
+      description: The query logs in Panorama.
+      script: '|||pan-os-query-logs'
+      type: regular
+      iscommand: true
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "73"
+    scriptarguments:
+      action:
+        simple: deny
+      extend-context:
+        simple: DetectedIPsFromSinkhole=response.result.log.logs.entry.src
+      ignore-outputs:
+        simple: "true"
+      log-type:
+        simple: traffic
+      number_of_logs:
+        simple: "5000"
+      polling:
+        simple: "true"
+      rule:
+        complex:
+          root: inputs.RuleNameToDenyToSinkhole
+          transformers:
+          - operator: SetIfEmpty
+            args:
+              applyIfEmpty: {}
+              defaultValue:
+                value:
+                  simple: RuleNameToDenyToSinkhole
+                iscontext: true
+      timeout:
+        simple: "180"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -240,
+          "y": 2390
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "77":
+    id: "77"
+    taskid: 2faf5424-b910-4eb7-8539-d46f8dffa8d4
+    type: condition
+    task:
+      id: 2faf5424-b910-4eb7-8539-d46f8dffa8d4
+      version: -1
+      name: Is PAN-OS enabled?
+      description: Returns 'yes' if integration brand is available. Otherwise returns 'no'
+      scriptName: IsIntegrationAvailable
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "73"
+      "yes":
+      - "85"
+    scriptarguments:
+      brandname:
+        simple: Panorama
+    results:
+    - brandInstances
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 450,
+          "y": 70
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "78":
+    id: "78"
+    taskid: b65e1e67-a89b-4e0f-8e2f-3cc18739d279
+    type: regular
+    task:
+      id: b65e1e67-a89b-4e0f-8e2f-3cc18739d279
+      version: -1
+      name: Search for existing rule denying traffic to sinkhole
+      description: Searches for rules that apply to traffic to the sinkhole address. Although it's recommended to deny traffic to the sinkhole address, because packets are dropped anyway, the search also looks for allow rules too. This is done because the goal is to log traffic to it, so the action is irrelevant.
```

Comment:
```suggestion
      description: Searches for rules that apply to traffic to the sinkhole address. Although it's recommended to deny traffic to the sinkhole address, because packets are dropped anyway, the search also looks for allow rules. This is done because the goal is to log traffic to it, so the action is irrelevant.
```